### PR TITLE
Allow `fwhm_fromlines` for tilts calibration

### PR DESCRIPTION
**Pull request by debora-pe**
As title. Addresses issue #1688 
Also, a small fix to `pypeit_chk_tilts`

### CODE REVIEW

File: pypeit/wavetilts.py [hunk 1]
```diff
@@ -268,24 +279,32 @@ class BuildWaveTilts:
         slits (:class:`~pypeit.slittrace.SlitTraceSet`):
             Slit edges
         spectrograph (:class:`~pypeit.spectrographs.spectrograph.Spectrograph`):
-            Spectrograph object
+            The `Spectrograph` instance that sets the instrument used.  Used to set
```
Comment (profxj) [inline pypeit/wavetilts.py:282]: thx for polishing these!

**Review by profxj — APPROVED** [review]
very good, thanks!

**Review by kbwestfall — APPROVED** [review]
Looks good to me!  Approving now, but we need to run the tests.

**Comment by debora-pe** [comment]
Tests pass (the failed unit tests are the usual `setup_gui` tests that my machine have problems with):
```
--- PYTEST PYPEIT UNIT TESTS PASSED  256 passed, 60 warnings in 363.78s (0:06:03) ---
--- PYTEST UNIT TESTS FAILED  7 failed, 145 passed, 168 warnings in 4220.26s (1:10:20) ---
--- PYTEST VET TESTS PASSED  62 passed, 99 warnings in 5440.19s (1:30:40) ---
--- PYPEIT DEVELOPMENT SUITE PASSED 247/247 TESTS  ---
Testing Started at 2024-10-21T17:38:51.617178
Testing Completed at 2024-10-22T07:48:40.394650
Total Time: 14:09:48.777472
```

**Pull request merged**